packages/dx-react-grid-bootstrap3/src/plugins/search-panel.jsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import { SearchPanel as SearchPanelBase } from '@devexpress/dx-react-grid';
+import { SearchPanelInput } from '../templates/search-panel-input';
+
+const defaultMessages = {
+  searchPlaceholder: 'Search...',
+};
+
+export const SearchPanel = ({ messages, ...restProps }) => (
+  <SearchPanelBase
+    inputComponent={SearchPanelInput}
+    messages={{ ...defaultMessages, ...messages }}
+    {...restProps}
+  />
+);
+
+SearchPanel.inputComponent = SearchPanelInput;
```

Comment:
Typically we don't use '...Component' suffix in static fields and start their names with a capital letter.
Also, I did not find this field description in the reference (typically we describe this in the 'Plugin Components' section).